### PR TITLE
front: show dust incidents in UI

### PR DESCRIPTION
**Pull request by spolu**
## Description

Fixes https://github.com/dust-tt/tasks/issues/1507

Show in the UI when we're having an incident. This will go a long way in making our users aware of ongoing issues.

Screenshot from 2024-10-18 16-01-12

## Risk

It does not work.

## Deploy Plan

- Secrets were updated https://dust4ai.slack.com/archives/C050SM8NSPK/p1729258285960699
- deploy `front`

### CODE REVIEW

**Comment by spolu** [comment]
PMRR it works on front-edge

Screenshot from 2024-10-18 16-41-08
Screenshot from 2024-10-18 16-41-05

File: front/components/navigation/NavigationSidebar.tsx [hunk 1]
```diff
@@ -184,18 +184,38 @@ export const NavigationSidebar = React.forwardRef<
...
 
-  return (
-    <div className="space-y-2 border-y border-pink-200 bg-pink-100 px-3 py-3 text-xs text-pink-900">
-      <div className="font-bold">{providerStatus.name}</div>
-      <div className="font-normal">{providerStatus.description}</div>
-    </div>
-  );
+  const { providersStatus, dustStatus } = appStatus;
+
+  if (dustStatus) {
```
Comment (flvndvd) [inline front/components/navigation/NavigationSidebar.tsx:194]: So we can only display one banner at a time?

**Review by flvndvd — COMMENTED** [review]
LGTM.

File: front/components/navigation/NavigationSidebar.tsx (hunk 1 above)
Comment (spolu) [reply front/components/navigation/NavigationSidebar.tsx:194]: dust has priority on providers. Only one at a time

**Pull request merged**